RFC-000-Process-description.md

```diff
@@ -0,0 +1,32 @@
+# Process description
+
+- RFC-Number: 000
+- Status: Draft
+- Discussion-issue: -
+- Created on: 25 Jan. 2019
+
+<!-- TOC GOES here -->
+
+## Description
+
+This document describes the RFC (Request-for-comments) process that is used to document and specify the COMIT procotol.
+The reason for using an RFC process for this is that we would like the documentation itself and the process around it to be open-source and accessible to anyone.
+
+## Lifecycle
+
+### Everything starts as `Draft`
+
+Anybody is welcome to submit an RFC to extend or change the COMIT protocol by opening a pull-request to this repository.
+All accepted changes will have status `Draft`.
+RFCs in status `Draft` MUST have a discussion issue that can be used to discuss the current state of the RFC.
+This will be the place where anybody can voice opinions or point out possible flaws or inconsistencies.
+As instructed by the [issue template](./.github/ISSUE_TEMPLATE/discussion_issue.md), please add a checklist for unresolved problems or open questions at the end of the discussion issue.
+As long as an RFC is in `Draft`, new PRs can change it accordingly to reflect the newest findings of the discussions.
+
+### On to stabilization
+
+As soon as all the contributors are happy with the RFC, a stabilization PR can be opened.
```

Comment:
❓ define happy? do we want to set a time period? (ie, no more comment has been made for one month)

Reply:
Good point. I'll make that an open question.

Reply:
@comit-network/rust-devs opinions?

Reply:
Yeah, that's pretty vague. I like the period without comment. I guess there's never gonna be a reason to rush an RFC out the door anyway.